oci8/oci8.php

```diff
@@ -5,6 +5,7 @@
 /**
  * OCI8 LOB functionality for large binary (BLOB) and character (CLOB) objects.
  * @link https://php.net/manual/en/class.OCI-Lob.php
+ * @removed 8.0
```

Comment:
```suggestion
 * @deprecated removed in 8.0
```
Would be better supported by tools and IDEs